model_compression_toolkit/exporter/model_wrapper/fw_agnostic/get_inferable_quantizers.py

```diff
@@ -32,18 +32,25 @@ def get_inferable_quantizers(node: BaseNode,
     Returns:
         weight_quantizers: A dictionary between a weight's name to its quantizer.
         activation_quantizers: A list of activations quantization, one for each layer output.
+        weight_values: A dictionary between a weight's name to its value. Relevant for positional weights only.
     """
 
     weight_quantizers = {}
     activation_quantizers = []
+    weight_values = None
 
     for attr in attributes_names:
         if node.is_weights_quantization_enabled(attr):
             weight_quantizer = get_weights_quantizer_for_node(node, attr)
             weight_quantizers[attr] = weight_quantizer
+            if isinstance(attr, int):  # for positional weights we need to extract the weight's value.
```

Comment:
Is it possible to do it in a separate function? It seems very weird the combination of returned values and I don't see why they should be coupled.